rundeckapp/grails-spa/packages/ui-trellis/src/app/components/home/HomeBrowserItem.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <div class="project_list_item" :key="`project${index}`" :data-project="project.name">
+    <div class="row row-hover row-border-top">
+      <div class="col-sm-6 col-md-8">
+        <ui-socket location="project-favorite" section="home" :socket-data="{projectName: project.name}" />
+        <a
+            :href="`/?project=${project.name}`"
+            class="link-hover text-inverse project_list_item_link link-quiet"
+            :id="`project${index}`"
+        >
+          <p class="h5">
+            {{ project.label ? project.label : project.name }}
+          </p>
```

Comment:
the `<p>` tag causes the name to be in a block element, and then the schedule/execution disabled indicator icon gets cleared to be below it.  I think it will need to be changed to a `<span>`

<img width="202" alt="Screenshot 2024-02-01 at 4 22 59 PM" src="https://github.com/rundeck/rundeck/assets/55603/2907216a-eb40-4126-a298-e3fecfe2782c">

Reply:
Would like to avoid having it as a span as it has no semantic value alone and it causes issues for accessibility (IIRC screen readers ignore orphan spans). Can just make the p have a different display value.